README.md

```diff
@@ -4,52 +4,55 @@ This box comes with everything you need to start using smart contracts from a re
 
 ## Installation
 
-1. Install Truffle and Ganache CLI globally. If you prefer, the graphical version of Ganache works as well!
-    ```javascript
-    npm install -g truffle
-    npm install -g ganache-cli
-    ```
+First ensure you are in a new and empty directory.
 
-2. Download the box. This also takes care of installing the necessary dependencies.
+1. Run the `unbox` command via `npx` and skip to step 3. This will install all necessary dependencies. A Create-React-App is generated in the `app` directory.
+   ```js
+   npx truffle unbox drizzle
```

Comment:
Hmm where's Truffle installed? I assume it's a local install of Truffle that this line refers to? Didn't see it in any of the `package.json`

Reply:
Truffle isn't installed explicitly, and neither are any of the boxes because we don't assume a Truffle project to always have a `package.json`.